openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -1187,6 +1188,7 @@
     "entity-fetch-error": "Error while fetching {{entity}}",
     "entity-fetch-version-error": "Error while fetching {{entity}} versions {{version}}",
     "entity-updating-error": "Error while updating {{entity}}",
+    "error-selected-node-name-details": "Error while getting {{selectedNodeName}} details",
```

Comment:
Can we make use of already existing key here `entity-fetch-error` instead of adding one more?

Reply:
@aniketkatkar97 It's not entity, We cannot use it